### PR TITLE
[zh] Update webhook.md

### PR DESCRIPTION
**Pull request by my-git9**
Update webhook.md

### CODE REVIEW

**Comment by tengqm** [comment]
/approve

Looks like the gate is broken.

**Comment by tengqm** [comment]
closed and reopen in attempt to trigger the gate.

File: content/zh/docs/reference/access-authn-authz/webhook.md [hunk 1]
```diff
@@ -266,6 +266,7 @@ to the REST api.
 
 <!--
 For further documentation refer to the authorization.v1beta1 API objects and
-[webhook.go](https://github.com/kubernetes/kubernetes/blob/{{< param "githubbranch" >}}/staging/src/k8s.io/apiserver/plugin/pkg/authorizer/webhook/webhook.go).
+[webhook.go](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/plugin/pkg/authorizer/webhook/webhook.go).
 -->
-更多信息可以参考 authorization.v1beta1 API 对象和[webhook.go](https://github.com/kubernetes/kubernetes/blob/{{< param "githubbranch" >}}/staging/src/k8s.io/apiserver/plugin/pkg/authorizer/webhook/webhook.go)。
+更多信息可以参考 authorization.v1beta1 API 对象和[webhook.go](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/plugin/pkg/authorizer/webhook/webhook.go).
+
```
Comment (ydFu) [inline content/zh/docs/reference/access-authn-authz/webhook.md:272]:
```suggestion
更多信息可以参考 authorization.v1beta1 API 对象和 [webhook.go](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/plugin/pkg/authorizer/webhook/webhook.go)。
```
Thank you for your contribution, please adjust the grammar of this sentence.

**Comment by ydFu** [comment]
/lgtm
Thank you.

**Pull request merged**